vm/mikrotik-routeros.sh

```diff
@@ -251,7 +251,7 @@ nfs | dir)
   DISK_REF="$VMID/"
   DISK_IMPORT="-format qcow2"
   ;;
-btrfs | zfspool)
+btrfs)
```

Comment:
can you add `zfspool` below this condition like this and test?
```
zfspool)
  DISK_EXT=""
  DISK_REF=""
  DISK_IMPORT="-format raw"
```

Reply:
Yes that works. Not sure what to do from here.

Reply:
How do you mean? If it works, add it to the PR and we will merge it :)

Reply:
I see you did the change in your `main` branch, but you need to add the change to your `patch-1` branch which this pull request is based on

Reply:
Thank you for your help with this, I've not used Git before. It does look like the change is in patch-1 branch now.

![image](https://github.com/user-attachments/assets/533aa0d6-7310-43c7-839b-5ba3a88ed5b8)